ui-core/src/components/inputs/Input.tsx

```diff
@@ -43,6 +44,8 @@ export type InputProps = React.InputHTMLAttributes<HTMLInputElement> &
     leadingContent?: InputAffixContent | InputAffixContentWithCallback;
     trailingContent?: InputAffixContent | InputAffixContentWithCallback;
     inputFieldWrapperClassname?: string;
+    showChevron?: boolean;
+    showClearButton?: boolean;
```

Comment:
I think the prop should be named `withClearButton` (true by default probably). And the clearButton shouldn't be displayed all the time but only when the input is not empty.